src/runtime/hexagon_host.cpp

```diff
@@ -918,6 +931,34 @@ WEAK void halide_hexagon_power_hvx_off_as_destructor(void *user_context, void *
     halide_hexagon_power_hvx_off(user_context);
 }
 
+WEAK int halide_hexagon_set_remote_thread_params(void *user_context, int priority, int stack_size) {
+    int result = init_hexagon_runtime(user_context);
+    if (result != 0) {
+        return result;
+    }
+
+    debug(user_context) << "halide_hexagon_set_remote_thread_params\n";
+    if (!set_remote_thread_params) {
+        // This runtime doesn't support changing the thread params.
+        return -1;
+    }
+
+    struct remote_rpc_thread_params th;
+    th.domain = CDSP_DOMAIN_ID;
+    th.prio = priority;
+    th.stack_size = stack_size;
+    debug(user_context) << "    set_remote_thread_params("
+                        << "thread_priority: " << priority
+                        << ", thread_stack_size: " << stack_size << ") -> \n";
+    result = set_remote_thread_params(FASTRPC_THREAD_PARAMS, (void *)&th, sizeof(th));
+    debug(user_context) << "        " << result << "\n";
+    if (result != 0) {
+        error(user_context) << "set_remote_thread_params faiiled.\n";
+        return result;
```

Comment:
Will this fastrcp error code not get conflated with a halide runtime error code?

Reply:
Yes, it will, this should return -1 or some other halide error code.